### ISSUE TITLE
Maybe allow user to decide on how to adjust

### ISSUE THREAD

**Issue by AMCalejandro** [neurogenomics/MAGMA_Celltyping#134]
https://github.com/neurogenomics/MAGMA_Celltyping/blob/8cb235fb495a467fb825819a76b2f154d3840146/R/calculate_celltype_associations.r#L136

Hi Brian,

Please, read here from the MAGMA manual    

![image](https://user-images.githubusercontent.com/84338703/201107804-eae0683e-8716-4a2d-b012-0412982d159f.png)

**Comment by bschilder** [comment]
I'm not really clear on what the request is here. Could you please explain what you're hoping for @AMCalejandro ?

Please use the **Feature Request** template.
https://github.com/neurogenomics/MAGMA_Celltyping/issues/new/choose

**Comment by AMCalejandro** [comment]
Yes, so maybe instead of assuming that I want to adjust with a Zscore, giving the user some freedom to decide whether to adjust with Zscore or the log of the P-value ,...

It has been a while, I think that is what I had in mind
